README.md

```diff
@@ -1 +1,25 @@
-# cheminformatics
\ No newline at end of file
+# cheminformatics
+
+-------------------
+
+Initial Setup:
+
+- clone this repo
+- create a conda environment with rdkit: `conda create -c rdkit -n my-rdkit-env rdkit`
+- install flask: `conda install flask`
+- `export FLASK_APP=cheminformatcis.py`
```

Comment:
typo in file name here

Reply:
Good catch, fixed! I also slightly updated the conda steps